components/weekoverview_components/activity_components/ActivityItem.tsx

```diff
@@ -49,7 +49,7 @@ const ActivityItem: React.FC<ActivityItemProps> = ({ time, isCompleted, setImage
         {data ? (
           <Pressable onPress={() => handleImagePress(data)}>
             <Image
-              source={{ uri: data }}
+              source={{ uri: data, cache: "force-cache" }}
```

Comment:
How much % performance increase is this? I think it's around 0% after reading docs, but I might be wrong

Reply:
anything over 0 is good 💯

Reply:
Why u bully me